internal/api/db_database_schema.go

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+//
+
+package api
+
+import (
+	"net/http"
+
+	"github.com/gorilla/mux"
+	"github.com/mattermost/mattermost-cloud/model"
+)
+
+// initDatabases registers database schema endpoints on the given router.
```

Comment:
```suggestion
// initDatabaseSchemas registers database schema endpoints on the given router.
```
We could also get rid of those comments for `init` methods as they do not provide much insight 0/5.